app/templates/ui/app/search/search-results.html

```diff
@@ -1,6 +1,6 @@
 <div ng-repeat="result in results">
   <h4>
-    <a ui-sref="root.view({uri: result.uri})">{{ result.uri }}</a>
+    <a ui-sref="root.view({uri: result.uri})">{{ result.title || result.uri }}</a>
```

Comment:
I vote you show me ... erm ... them how to supply a title (maybe just a comment here, or maybe a recipe in the docs?)